opennms-webapp/src/main/webapp/admin/userGroupView/users/modifyUser.jsp

```diff
@@ -450,6 +453,29 @@
             <input class="form-control" type="text" id="homePhone" name="homePhone" value='<%=(homePhone == null ? "":homePhone)%>'/>
           </div>
         </div>
+
+    <div class="form-group">
+          <label for="timeZoneId" class="col-sm-2 control-label">Time Zone:</label>
```

Comment:
Can we move this under a new `<h3>` tag? Right now it looks like it's part of the `Notification Information`.

Maybe add a new section called `Date/Time Preferences`?